advanced_guides/asynchronous_updates.md

```diff
@@ -25,21 +25,21 @@ sequenceDiagram
   M->>-Q: dequeue update 2
 </mermaid>
 
-### Which actions are async?
+### Which updates are async?
 
-Every action which could be compute-expensive is asynchronous. These include:
+Every update which could be compute-expensive is asynchronous. These include:
 - Create/update a schema
 - Update index settings
 - Add/update/delete documents
 
 ### Understanding updates
 
 Updates returns the following information:
-* **status**: State of the action (enqueued, processed)
+* **status**: State of the update (enqueued, processed)
 * **updateId**: Id of the update
-* **type**: Information about the action type
-* **enqueuedAt**: Date at which the action has been added to the queue
-* **processedAt**: Date ate which the action has done processing.
+* **type**: Information about the update type
+* **enqueuedAt**: Date at which the update has been added to the queue
+* **processedAt**: Date ate which the update has done processing.
```

Comment:
same as above: operation, actions or process

Reply:
```suggestion
* **processedAt**: Date at which the operation has done processing.
```

Reply:
This suggestion is good